docs/playground/src/Playground.ts

```diff
@@ -68,7 +68,13 @@ export class Playground {
 
   public initializeComponent() {
     let configuration = this.getConfiguration();
-    SearchEndpoint.configureSampleEndpointV2();
+    // `@hierarchicfield` does not exist in the sample Coveo Cloud V2 organization.
```

Comment:
I think this can be done instead in `PlaygroundConfiguration.ts` by adding a `toExecute` function for that component (which replaces the endpoint)